PWGHF/D2H/TableProducer/dataCreatorCharmResoReduced.cxx

```diff
@@ -134,14 +138,35 @@ struct HfDataCreatorCharmResoReduced {
   o2::base::Propagator::MatCorrType matCorr = o2::base::Propagator::MatCorrType::USEMatCorrLUT;
   HfHelper hfHelper;
   o2::hf_evsel::HfEventSelection hfEvSel;
+  o2::vertexing::DCAFitterN<2> fitter;
+  double bz{0.};
 
   bool isHfCandResoConfigFilled = false;
 
+  // Helper struct to pass V0 informations
+  struct {
+    std::array<float, 3> pos;
+    std::array<float, 3> mom;
+    std::array<float, 3> momPos;
+    std::array<float, 3> momNeg;
+    float pT;
+    float cosPA;
+    float dcaV0ToPv;
+    float dcaDau;
+    float alpha;
+    float eta;
+    float radius;
+    float mK0Short;
+    float mLambda;
+    uint8_t v0Type;
+  } candidateV0;
+
   using CandsDplusFiltered = soa::Filtered<soa::Join<aod::HfCand3Prong, aod::HfSelDplusToPiKPi>>;
   using CandsDplusFilteredWithMl = soa::Filtered<soa::Join<aod::HfCand3Prong, aod::HfSelDplusToPiKPi, aod::HfMlDplusToPiKPi>>;
   using CandDstarFiltered = soa::Filtered<soa::Join<aod::HfD0FromDstar, aod::HfCandDstars, aod::HfSelDstarToD0Pi>>;
   using CandDstarFilteredWithMl = soa::Filtered<soa::Join<aod::HfD0FromDstar, aod::HfCandDstars, aod::HfSelDstarToD0Pi, aod::HfMlDstarToD0Pi>>;
   using TracksWithPID = soa::Join<aod::Tracks, aod::TracksCov, aod::TracksExtra, aod::TrackSelection, aod::TracksDCA, aod::pidTPCFullPi, aod::pidTOFFullPi, aod::pidTPCFullKa, aod::pidTOFFullKa, aod::pidTPCFullPr, aod::pidTOFFullPr>;
+  using TracksIUWithPID = soa::Join<aod::TracksIU, aod::TracksExtra, aod::TracksCovIU, aod::pidTPCPi, aod::pidTPCPr>;
```

Comment:
```suggestion
  using TracksIUWithPID = soa::Join<aod::TracksIU, aod::TracksExtra, aod::TracksCovIU, aod::pidTPCPi, aod::pidTPCPr>;
```
I would add the possibility to apply a cut also on the NsigmaTOF (applied in the trigger for the proton from Lambda), so we need to consume the TOF tables

Reply:
Done!